.nvmrc

```diff
@@ -1 +1 @@
-v18
\ No newline at end of file
+18
```

Comment:
Can you revert this to be back `v18` as is been orginally??

Reply:
done